src/loader-service.ts

```diff
@@ -0,0 +1,35 @@
+import type { PodcastEpisode, PodcastSeries } from './podcast';
+
+export async function getPodcast(id: string) {
+  console.log('loading podcast', id);
```

Comment:
suggestion (non-blocking): perhaps we could leave a TODO comment here? I find that they nicely delineate the code that's done from the code that should be returned to.